src/Helpers/Form/Utils.php

```diff
@@ -1,13 +1,12 @@
 <?php
 namespace Give\Helpers\Form\Utils;
 
-/**
- * Get result if we are viewing embed form or not
- *
- * @since 2.7.0
- *
- * @return bool
- */
+use WP_Post;/**
```

Comment:
Please correct this formatting.

Reply:
fixed: https://github.com/impress-org/givewp/compare/de50ef46885f...349f67dcf3fc